src/quick-lint-js/fe/parse-class.cpp

```diff
@@ -1155,6 +1155,8 @@ void Parser::parse_and_visit_class_or_interface_member(
     void check_modifiers_for_method() {
       error_if_accessor_method();
       error_if_declare_method();
+      error_if_generator_method();
+      error_if_async_method();
```

Comment:
Nit: I think "error if async method" is a misleading name. "error if async getter or setter" is more appropriate.
```suggestion
      error_if_generator_getter_or_setter();
      error_if_async_getter_or_setter();
```